libgalois/include/katana/PropertyGraphRetractor.h

```diff
@@ -69,10 +69,30 @@ class KATANA_EXPORT PropertyGraphRetractor {
     return pg_->rdg_.edge_properties();
   }
 
+  /// Return true if type information has been loaded separate from properties.
+  /// Return false otherwise.
+  bool IsEntityTypeIDsOutsideProperties() {
```

Comment:
We could simplify this name to `hasEntityTypeIDs` or `hasValidEntityTypeIDs`

Reply:
I would love to. Can I change it in `RDG` and friends as well?

Reply:
Right now I am going with `NeedsEntityTypeIDInference()`

Reply:
That's OK too.